platform-common-testing/src/main/java/com/softicar/platform/common/testing/Asserts.java

```diff
@@ -65,6 +66,16 @@ public static void assertThrows(Class<? extends Throwable> expectedThrowableClas
 			expectedThrowableClass.isAssignableFrom(thrownClass));
 	}
 
+	public static void assertExceptionMessage(IDisplayString expectedMessage, INullaryVoidFunction thrower) {
+
+		try {
+			thrower.apply();
+			fail("missing exception");
```

Comment:
```suggestion
			fail("An expected Exception failed to occur.");
```

Reply:
well, a little bit too verbose of my taste, but did it nevertheless ;)